src/components/access-settings/AccessSettings.tsx

```diff
@@ -57,13 +58,8 @@ const useStyles = makeStyles((theme: Theme) => ({
   error: {},
 }))
 
-function getNameDisplay({
-  firstName,
-  lastName,
-
-  synapseUserId,
-}: LoggedInUserData): string {
-  const name = firstName || lastName ? [firstName, lastName].join(' ') : synapseUserId
+function getNameDisplay({firstName, lastName, email, synapseUserId}: LoggedInUserData): string {
+  const name = firstName || lastName ? [firstName, lastName].join(' ') : synapseUserId || email
```

Comment:
Fallback to email address when member doesn't have a first/last name or a synapseUserId